subgraphs/erc20/schema.graphql

```diff
@@ -0,0 +1,293 @@
+# Subgraph Schema: ERC20 Tokens
+# Version: 1.0.0
+
+#############################
+####### Token Metadata ######
+#############################
+
+type Token @entity {
+  " Smart contract address of the token "
+  id: ID!
+
+  " Name of the token, mirrored from the smart contract "
+  name: String!
+
+  " Symbol of the token, mirrored from the smart contract "
+  symbol: String!
+
+  " The number of decimal places this token uses, default to 18 "
+  decimals: Int!
+
+  " If token transfers are paused, derived from paused event "

```

Comment:
`pause` doesn't seem to be a standard ERC20 event. What are some tokens that have `pause` event?

Reply:
yeah, it's not a standard ERC20 event, I have removed this from the schema